backend/cfg/sub_cfg.mli

```diff
@@ -64,9 +64,9 @@ val iter_basic_blocks : t -> f:(Cfg.basic_block -> unit) -> unit
 
 val exists_basic_blocks : t -> f:(Cfg.basic_block -> bool) -> bool
 
-val join : from:t list -> to_:t -> t
+val join : from:t list -> to_:t -> unit
 
-val join_tail : from:t list -> to_:t -> t
+val join_tail : from:t list -> to_:t -> unit
```

Comment:
(Maybe change a label to make it
clear "which" is modified? That
could be `to_` ~> `into`?)

Reply:
Noted, I will do this in the next PR to avoid yet another CI run